owid/walden/index/dummy/2020/dummy.json

```diff
@@ -0,0 +1,16 @@
+{
+  "namespace": "dummy",
+  "short_name": "dummy",
+  "name": "Dummy dataset",
+  "description": "This\nis\na\ndummy\ndataset\n",
+  "source_name": "dummy source",
+  "url": "https://www.dummy.com/",
+  "file_extension": "xlsx",
+  "date_accessed": "2022-08-19",
+  "source_data_url": "https://www.rug.nl/ggdc/historicaldevelopment/maddison/data/mpd2020.xlsx",
+  "is_public": true,
+  "version": "2020",
```

Comment:
I think it is in general better to have a date instead of a year, for `version`. Otherwise, if we have to update the dataset again during the same year, we may have a problem.